alpha/veneer/semver/semver.go

```diff
@@ -434,3 +445,41 @@ func MermaidChannelWriter(cfg declcfg.DeclarativeConfig, out io.Writer) error {
 	}
 	return nil
 }
+
+func withoutBuildMetadataConflict(versions *map[string]semver.Version) error {
```

Comment:
Do we need to pass a pointer here if we aren't manipulating that variable anywhere in this function? It's a shame Go doesn't have const refing primitives in the language.

Reply:
IKR?  I was being really conservative and passing the minimum necessary overhead.  You let me know if that's a concern.